week 1/lists_n_loops/readme.md

```diff
@@ -0,0 +1,22 @@
+Python lists can be iterated through different means, which include but not limited to:
+
+-> simple for loop
+-> for loop with range
+-> while loop
+-> enumerate
+-> list comprehension
```

Comment:
use markdown list format. If you are using PyCharm/VS code, you can see the render of this md file locally.